etna/lib/etna/clients/base_client.rb

```diff
@@ -29,7 +29,8 @@ def token_expired?
       def token_will_expire?(offset=3000)
         # offset in seconds
         # Will the user's token expire in the given amount of time?
-        epoch_seconds = JSON.parse(Base64.urlsafe_decode64(token.split('.')[1]))["exp"]
+        payload = JSON.parse(Base64.urlsafe_decode64(token.split('.')[1]))
```

Comment:
This non functional change just made it easier for me to debug by examing the payload during a test.